src/planner/partialize.c

```diff
@@ -660,7 +660,7 @@ get_best_total_path(RelOptInfo *output_rel)
 /*
  Is the provided path a agg path that uses a sorted or plain agg strategy?
 */
-static bool
+static bool pg_nodiscard
 is_path_sorted_or_plain_agg_path(Path *path)
```

Comment:
In a follow-up task, we should try to remove this function entirely. Instead, we should create a sorted and hashed partial aggregation finalizer node and allow PostgreSQL to choose the finalizer strategy based on costs (this needs some testing to avoid regregressions).